packages/webvh/src/anoncreds/services/WebVhAnonCredsRegistry.ts

```diff
@@ -103,7 +104,8 @@ export class WebVhAnonCredsRegistry implements AnonCredsRegistry {
       agentContext.config.logger.trace(`Resource ${resourceId} attestation found.`)
 
       // 2. Check for valid proof
-      const isProofValid = await this.verifyProof(agentContext, resourceObject.proof, resourceObject.content)
+      const unsecuredDocument = (({ proof, ...o }) => o)(resourceObject);
```

Comment:
Isn't this the same as

```suggestion
      const { proof, ...unsecuredDocument } =  resourceObject
```

Reply:
applied suggestion, thanks